Backoffice/Form/Type/BlockType.php

```diff
@@ -76,6 +79,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'group_id' => 'technical',
             'sub_group_id' => 'cache',
         ));
+        if (
+            isset($options['data']) &&
+            $options['data'] instanceof BlockInterface &&
+            $options['data']->isTransverse()
+        ) {
+            $builder->add('code', 'text', array(
+                'label' => 'open_orchestra_backoffice.form.block.code',
+                'required' => false,
+                'group_id' => 'technical',
+                'sub_group_id' => 'code',
+                'constraints' => new UniqueBlockCode(array(
```

Comment:
What about having this constraint on entity despite form property and bubble the error?